docs/website/root/mithril/advanced/mithril-protocol/certificates.md

```diff
@@ -1,9 +1,9 @@
 ---
 sidebar_position: 2
-sidebar_label: Certificate chain in depth
+sidebar_label: Certificate chain design
 ---
 
-# Mithril certificate chain in depth
+# Certificate chain design
 
 ## Introduction
```

Comment:
Suggest dropping the first paragraph as the protocol is explained in the above sections. 
Suggest starting with 'The **certificate chain** is a Mithril component that...'

Reply:
Done :+1: